tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes-update.php

```diff
@@ -40,10 +40,12 @@ public function it_should_not_translate_if_post_is_source() {
 
 	/**
 	 * @test
+	 * @group wpmlcore-5834
 	 */
 	public function it_should_translate() {
 		$target_lang = 'fr';
 		$source_lang = 'en';
+		$original_post_id   = mt_rand( 1, 100 );
```

Comment:
When Andrea reviewed my first commits, he asked not to use random numbers/strings in tests, not to guess after why tests are not running unexpectedly.  I do not insist, of course )